src/main/java/org/openstreetmap/atlas/checks/flag/FlaggedRelation.java

```diff
@@ -0,0 +1,112 @@
+package org.openstreetmap.atlas.checks.flag;
+
+import java.util.Map;
+
+import org.openstreetmap.atlas.geography.Location;
+import org.openstreetmap.atlas.geography.Rectangle;
+import org.openstreetmap.atlas.geography.atlas.items.Relation;
+import org.openstreetmap.atlas.geography.atlas.items.RelationMemberList;
+import org.openstreetmap.atlas.tags.RelationTypeTag;
+import org.openstreetmap.atlas.tags.annotations.validation.Validators;
+
+import com.google.gson.JsonObject;
+
+/**
+ * A flag for a {@link Relation}
+ *
+ * @author sayas01
+ */
+public class FlaggedRelation extends FlaggedObject
+{
+    private final Relation relation;
+    private final Map<String, String> properties;
+
+    public FlaggedRelation(final Relation relation)
+    {
+        this.relation = relation;
+        this.properties = initProperties(relation);
+    }
+
+    /**
+     * As relations itself do not have a geometry, the geometry of FlaggedRelation is set to null.
+     *
+     * @return flagged geometry
+     */
+    @Override
+    public Iterable<Location> getGeometry()
+    {
+        return null;
+    }
+
+    /**
+     * @return flag key-value property map
+     */
+    @Override
+    public Map<String, String> getProperties()
+    {
+        return this.properties;
+    }
+
+    /**
+     * A GeoJSON representation of the flagged object.
+     *
+     * @param flagIdentifier
+     *            We always will want to know the id of the flag assocaited with this flag object.
+     * @return GeoJSON representation of the flagged object.
+     */
+    @Override
+    public JsonObject asGeoJsonFeature(final String flagIdentifier)
+    {
+
+        final JsonObject feature = this.relation.asGeoJsonFeature();
+        final JsonObject properties = feature.getAsJsonObject("properties");
+
+        properties.addProperty("flag:id", flagIdentifier);
```

Comment:
Is this necessary ? Each object will already contain, `identifier`, `osmid` and `itemType`

Reply:
This was added to match the "properties" of FlaggedPoint and FlaggedPolyLine. I can take it down for now. But the FlaggedPoint and FlaggedPolyLine properties will have these additional elements.

Reply:
Hmm, super interesting. I wonder why? For consistency sake, I guess its a good idea to keep it

Reply:
I am also not sure as the information is redundant. I will add these back.